kotlinx.fuzz.test/build.gradle.kts

```diff
@@ -13,11 +15,14 @@ repositories {
 dependencies {
     testImplementation(kotlin("test")) // adds green arrow in IDEA (no idea why)
     testRuntimeOnly("kotlinx.fuzz:kotlinx.fuzz.jazzer")
+    implementation("org.jetbrains.kotlinx:kotlinx-serialization-protobuf:1.7.3")
 }
 
 fuzzConfig {
+    keepGoing = 10
+    runMode = RunMode.REGRESSION
     instrument = listOf("kotlinx.fuzz.test.**")
-    maxSingleTargetFuzzTime = 10.seconds
+    maxSingleTargetFuzzTime = 3600.seconds
```

Comment:
Also, I'm not sure that it is a good idea to set a 1hr timeout for each test in the `kotlinx.fuzz.test` in CI. It is a good test, and it is good to run it from time to time. But overall it will increase our CI feedback cycle and we will just waste a lot of computation time

Reply:
Fixed